README.md

```diff
@@ -197,7 +218,7 @@ connection.release(preparedStatement: preparedStatement) { result in
 ### Table creation
 Swift-Kuery enables you to create tables on the database server.
 
-Let's rewrite our `Grades` table by adding columns type and constraints: 
+Let's rewrite our `Grades` table by adding columns type and constraints:
```

Comment:
Let's revisit the `Grades` table, which we used in our Example above, and add in two new columns containing the data type and contraints.